version1/02 - Object types.md

```diff
@@ -90,863 +94,615 @@ The following notation is used in the tables below:
-
-
-### Korrespondansepart
-
-| Feild no. | Type      | Field                         | Not null | Has default | Code list              |  | Save | View | Query | Sort | Link | Unlink | Comment                                           |
+|           | ref       | refDokumentflyt                   |          |             |                    |  |      |      |   V   |      |  V   |        |                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|           | ref       | refAvskrivning                    |          |             |                    |  |      |      |   V   |      |  V   |        |                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|           | ref       | refTilknyttetAvskrivning          |          |             |                    |  |      |      |   V   |      |  V   |   V    |                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|           | ref       | refPresedens                      |          |             |                    |  |      |      |   V   |      |  V   |   V    | refers to **Presedens**                                                                                                                                                                                                                                                                                                                                                                                         |
+
+NOTE: journalansvarlig and journalansvarligBrukerIdent are inherited from the Registry Entry type and re-introduced in the table above because of the added constraints
```

Comment:
```suggestion
NOTE: journalansvarlig and journalansvarligBrukerIdent are inherited from the Basic Record type and re-introduced in the table above because of the added constraints.
```